data/templates/install-ubuntu/ubuntu-preseed

```diff
@@ -166,28 +177,32 @@ d-i finish-install/reboot_in_progress note
 xserver-xorg xserver-xorg/autodetect_monitor boolean true
 #### Advanced options
 ### Running custom commands during the installation
-# d-i pretemplateing is inherently not secure. Nothing in the installer checks
+# d-i preseeding is inherently not secure. Nothing in the installer checks
```

Comment:
I identify this should be a typo, this doesn't match with the official example, previous developer should have done some global replace operation (replace seed with template).